src/docfx/lib/git/CommitBuildTimeProvider.cs

```diff
@@ -40,6 +33,7 @@ public void Save()
 
         using (PerfScope.Start($"Saving commit build time for {_repo.Commit}"))
         {
+            _buildTimeByCommit = ReadLatestCacheIfAny();
```

Comment:
Should we lock on this operation (read + write)? Otherwise, the file might be read by two providers but only last data are saved.